tests/filesystem/test_cloud.py

```diff
@@ -0,0 +1,110 @@
+import boto3
+from tamr_toolbox.filesystem import cloud
+import pytest
+import google
+import botocore
+import tempfile
+from unittest.mock import patch
+
+
+def test_gcs_download():
+    @patch("google.cloud.client.Client")
+    def test_download(mock_client):
+        client = mock_client()
+
+        with tempfile.NamedTemporaryFile() as download_tmp:
+            cloud.gcs_download(
+                cloud_client=client,
+                source_filepath="path_in_bucket.txt",
+                destination_filepath=download_tmp.name,
+                bucket_name="test-bucket",
+            )
```

Comment:
These tests are not actually checking that the function does anything, only that it doesn't raise errors.  You can take them further by asserting that the underlying package's functions are being called like this:
```suggestion
            )
            
            bucket = client.get_bucket
            bucket.assert_called_with("test-bucket")

            blob = bucket().blob
            blob.assert_called_with("path_in_bucket.txt")

            download = blob().download_to_filename
            download.assert_called_with(download_tmp.name)
```

Please also make similar changes to the other 3 tests.